### PR TITLE
[10.0][MIG] fetchmail_notify_error_to_sender

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Standard migration from v9.

### CODE REVIEW

**Review by LoisRForgeFlow — APPROVED** [review]
code review :+1:

**Comment by MiquelRForgeFlow** [comment]
@Yajo, the milestone you put is wrong  😉

**Comment by yajo** [comment]
oops! 😅

**Comment by eLBati** [comment]
Hi @mreficent did you also test it?

I ported this module to 10 too and remember simple changes like https://github.com/OCA/server-tools/pull/902/commits/0336c9c7f35bac59be2f9ea08d933fd97a595210 were not enough.

Indeed, my porting commit is https://github.com/OCA/server-tools/pull/904/commits/a87e1b39245d05f59c04f435423a1cf9f94290da

Can you check?

Thanks

**Comment by MiquelRForgeFlow** [comment]
@eLBati , are you fine if I cherry-pick your commit?

**Comment by eLBati** [comment]
@mreficent :+1:

**Comment by MiquelRForgeFlow** [comment]
Can everyone update review? 🙂

File: fetchmail_notify_error_to_sender/README.rst [hunk 1]
```diff
@@ -1,38 +1,63 @@
...
+Images
+------
+
+* Odoo Community Association: `Icon <https://github.com/OCA/maintainer-tools/blob/master/template/module/static/description/icon.svg>`_.
+
 Contributors
 ------------
 
 * Lorenzo Battistini <lorenzo.battistini@agilebg.com>
+* Miquel Raïch <miquel.raich@eficent.com> (migration to v9)
```
Comment (sebalix) [inline fetchmail_notify_error_to_sender/README.rst]: and v10 ?

**Review by sebalix — APPROVED** [review]
Code review only, just a little comment about the contributors :+1:

**Pull request merged**